plugin/etcd/README.md

```diff
@@ -172,15 +172,45 @@ dig +short skydns.local @localhost
 1.1.1.2
 ~~~
 
+### Zone name as AAAA record
+
 If you would like to use `AAAA` records for the zone name too, you can set the following:
 ~~~
 % etcdctl put /skydns/local/skydns/x3 '{"host":"2003::8:1","ttl":"60"}'
 % etcdctl put /skydns/local/skydns/x4 '{"host":"2003::8:2","ttl":"60"}'
 ~~~
 
-If you query the zone name now for `AAAA` now, you will get the following response:
+If you query the zone name for `AAAA` now, you will get the following response:
 ~~~ sh
 dig +short skydns.local AAAA @localhost
 2003::8:1
 2003::8:2
 ~~~
+
+### SRV record
+
+If you would like to use `SRV` records, you can set the following:
+~~~
+%etcdctl put /skydns/local/skydns/srv '{"host":"skydns-local.server","ttl":60,"priority":10,"port":8080}'
+~~~
+Please notice that the key `host` is the `target` in `SRV`, so it should be a domain name.
+
+If you query the zone name for `SRV` now, you will get the following response:
+
+~~~ sh
+dig +short skydns.local SRV @localhost
+10 100 8080 skydns-local.server.
+~~~
+
+### TXT record
+
+If you would like to use `TXT` records, you can set the following:
+~~~
+%etcdctl put /skydns/local/skydns/txt '{"ttl":60,"text":"this is a random text message."}'
```

Comment:
would use /txt as that looks confusing

Reply:
Do you mean that the '/txt' in the '/skydns/local/skydns/txt' looks confusing? Yes, maybe I should change it to a random word like 'x5' or something like what we have for the AAAA and A records. And also for SRV example, using another one instead of using /srv.

Reply:
Done